tests/playwright/src/specs/kind.spec.ts

```diff
@@ -35,8 +35,8 @@ const EXTENSION_LABEL: string = 'podman-desktop.kind';
 const CLUSTER_NAME: string = 'kind-cluster';
 const KIND_CONTAINER_NAME: string = `${CLUSTER_NAME}-control-plane`;
 const CLUSTER_CREATION_TIMEOUT: number = 200000;
-
 let resourcesPage: ResourcesPage;
+
```

Comment:
Why did this new line move? :D

Reply:
I fiddled around with the variable at some point and I guess the formatter liked it like this better when I added it back.